utils/nonNull.ts

```diff
@@ -0,0 +1,48 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Registry } from 'azure-arm-containerregistry/lib/models';
+import { Subscription } from 'azure-arm-resource/lib/subscription/models';
+import { isNullOrUndefined } from 'util';
```

Comment:
Looks like this function could be housed here. Your thoughts?

Reply:
Given that this is a Node.js internal function, that would be pretty hard.  :-)

Reply:
To be honest, I had exactly the same thought when reviewing my own code... :-)